.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,44 @@
+---
+name: Bug Report
+about: 'Create a report to help us improve. '
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+<!-- 
+Read the guidelines for filing an issue first. 
+
+https://github.com/microsoft/python-language-server/blob/master/TROUBLESHOOTING.md#filing-an-issue
```

Comment:
It might be a good idea to grab some specifics from this guide and duplicate them here, i.e. above the log placeholder, say how to get trace logging, etc.

Reply:
I am worried about having duplicate parts because then we will have to update both in case, what do you think?

Reply:
The troubleshooting having an issue filing section is really just a stopgap before having these real templates. The best case would be to have it pared down to only long form stuff that doesn't feel right in the templates.

Reply:
ok I'll add it in